app/layout.tsx

```diff
@@ -18,11 +18,13 @@ export default function RootLayout({ children }: { children: React.ReactNode })
       </head>
       <body className="light font-sans">
         <Providers>
-          <LanguageProvider>
-            <div className="flex flex-col h-screen bg-background">
-              <Header />
-              <main className="flex-grow">{children}</main>
-            </div>
```

Comment:
maybe you should leave line 21-25 like it is and not change to your version. It may cause you conflict when you are merging to develop. So if you change back the code as it looked like before (the red area) then it will be better for you I think even though the code is the same but with another structure.

Reply:
Ok I check it out!